sdk/digitaltwins/Microsoft.Azure.Management.DigitalTwins/src/Generated/Models/GroupIdInformation.cs

```diff
@@ -29,10 +29,11 @@ public GroupIdInformation()
         /// <summary>
         /// Initializes a new instance of the GroupIdInformation class.
         /// </summary>
+        /// <param name="properties">The group information properties.</param>
         /// <param name="id">The resource identifier.</param>
         /// <param name="name">The resource name.</param>
         /// <param name="type">The resource type.</param>
-        public GroupIdInformation(GroupIdInformationPropertiesModel properties, string id = default(string), string name = default(string), string type = default(string))
```

Comment:
Breaking change

Reply:
Thanks! This was not intended to be a breaking change. I wonder if this stems from AutoFac generation, or it somehow slipped through the Swagger validation...

Reply:
Yes. It looks like there is a refactoring in the swagger definition, which causes the breaking change. Normally that should be caught by swagger linter.

Reply:
I've added GroupIdInformationPropertiesModel that inherits from GroupIdInformationProperties. Hopefully that resolves the issue.